src/components/navigation/navigation.spec.js

```diff
@@ -291,13 +291,17 @@ describe('script: navigation', () => {
   );
 });
 
-describe.each([['sub', EXAMPLE_NAVIGATION_WITH_SUBNAVIGATION_REMOVED, '.ons-navigation--sub']])(
-  'level: %s navigation',
-  (_, params) => {
-    it('removes the sub-nav element when hideHorizontalSubNav is set to true', async () => {
-      await setTestPage('/test', renderComponent('header', params));
+describe.each([['sub']])('level: %s navigation', () => {
```

Comment:
```suggestion
describe.each()('level: sub navigation', () => {
```
Another change we can make here to simplify things